Gems/Atom/RPI/Code/Source/RPI.Public/RenderPipeline.cpp

```diff
@@ -6,7 +6,7 @@
  *
  */
 
-
+#include "Atom/RPI.Public/Base.h"
```

Comment:
#include <Atom/RPI.Public/Base.h>

and move it to above this line
#include <Atom/RPI.Public/Pass/PassFilter.h>

Reply:
done